mailu/README.md

```diff
@@ -14,24 +14,37 @@
+| `mail.messageSizeLimitInMegabytes` | Message size limit in Megabytes                                                                                                   | `50`                                                           |
+| `mail.authRatelimit`               | Rate limit for authentication requests                                                                                            | `10/minute;1000/hour`                                          |
+| `initialAccount.username`          | Local part (part after @) for initial admin account                                                                               | not set                                                        |
+| `initialAccount.domain`            | Domain part (part before @) for initial admin account                                                                             | not set                                                        |
+| `initialAccount.password`          | Password for initial admin account                                                                                                | not set                                                        |
+| `certmanager.issuerType`           | Issuer type for cert manager                                                                                                      | `ClusterIssuer`                                                |
+| `certmanager.issuerName`           | Name of a preconfigured cert issuer                                                                                               | `letsencrypt`                                                  |
+| `persistence.size`                 | requested PVC size                                                                                                                | `100Gi`                                                        |
+| `persistence.type`                 | type of persistence (`hostPath` or `existingClaim`)                                                                               | `hostPath`                                                     |
+| `persistence.hostPath`             | path of the hostPath persistence                                                                                                  | not set                                                        |
+| `persistence.storageClass`         | storageClass to use for persistence (mutually exclusive with `hostPath` choice)                                                   | not set                                                        |
+| `persistence.existingClaim`        | existing PVC (mutually exclusive with `hostPath`, mandatory for `existingClaim` persistence type)                                 | not set                                                        |
```

Comment:
Please check why the whole table has changed.

Reply:
It's my linter that did that.
The whitespace has changed so that the table is also aligned in the editor.
Do you want me to fix it?

I've also added the new parameters.

Reply:
No, it's ok when it's in sync with current master (so that we don't loose anything there)